mona_sdk/client.py

```diff
@@ -67,6 +67,12 @@
     os.environ.get("MONA_SDK_WAIT_TIME_FOR_AUTHENTICATION_RETRIES_SEC", 2)
 )
 
+# When this variable is True, failed messages (for any reason) will be logged at "Error"
+# level.
+SAVE_FAILED_MESSAGES_LOGS = get_boolean_value_for_env_var(
```

Comment:
I think the word SAVE is not the right one here. Should be LOG, no? or SHOULD_LOG?